crates/codegen/parser/runtime/src/query_parser_tests.rs

```diff
@@ -0,0 +1,26 @@
+use super::query_model::Query;
+
+fn run_parser_test(input: &str, result: &str) {
+    assert_eq!(Query::parse(input).unwrap().unparse(), result);
+}
+
+#[test]
+fn test_parsing() {
+    run_parser_test(
```

Comment:
I wonder about the long term plan for such tests .. will they use something like `cst_snapshots` to write `generated/*` files?

If not, in order to assert the actual model parsed, what do you think about using something like [inline snapshots](https://docs.rs/insta/latest/insta/#inline-snapshots) to generate/update them in the same file?

Reply:
I don't see the benefit in using snapshots here, I think traditional tests are fine, especially, but not only because  we don't need many tests here because the engine tests also coincidentally test the parser.